tool/lib/commands/update_flutter_sdk.dart

```diff
@@ -133,35 +134,28 @@ class UpdateFlutterSdkCommand extends Command {
       repo.toolDirectoryPath,
       flutterSdkDirName,
     );
-    final toolFlutterSdk = Directory.fromUri(Uri.file(toolSdkPath));
+    final toolFlutterSdk = Directory(toolSdkPath);
```

Comment:
nit: `toolFlutterSdkDirectory`? (since we have a `FlutterSdk` class and this isn't an instance of that)

(or maybe just `toolSdkDirectory` to be consistent with `toolSdkPath`)